LiteDB.Tests/Issues/Issue2487_Tests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FluentAssertions;
+
+using Xunit;
+
+namespace LiteDB.Tests.Issues;
+
+public class Issue2487_tests
+{
+    private class DataClass
+    {
+        [BsonId]
+        public int Id { get; set; }
+
+        public string Foo { get; set; }
+
+        public string Bar { get; set; }
+    }
+
+    [Fact]
+    public void Test_Contains_EmptyStrings()
+    {
+        using var engine = new ConnectionString(":memory:").CreateEngine();
+
+        using var db = new LiteDatabase(engine);
+        var collection = db.GetCollection<DataClass>("data");
+
+        collection.Insert(new DataClass { Foo = "bar", Bar = "abc" });
```

Comment:
can you add a case where the value is and `string.Empty` as well? Then I think it should be good to be merged

Reply:
You mean that one DataClass.Foo is string.Empty?

Reply:
Yes, having values that are, null; empty